integrations-service/integrations/utils/integrations/remote_browser.py

```diff
@@ -231,6 +231,16 @@ async def press_key(self, key_combination: str) -> RemoteBrowserOutput:
         """Press a key or key combination"""
         # Split combination into individual keys
         keys = key_combination.split("+")
+        keys = [
```

Comment:
Why is this needed?

Reply:
Because the key mappings of computer use are different from the ones used by playwright, would you rather we handle that in browser-control and not here?

Reply:
No this is fine